std/js/src/main/scala/cats/effect/std/Console.scala

```diff
@@ -32,18 +32,16 @@ import java.nio.charset.Charset
  *
  * @example
  *   {{{
- * import cats.effect.std.Console
- * import cats.effect.kernel.Sync
- * import cats.syntax.all._
+ *  import cats.Monad
+ *  import cats.effect.std.Console
+ *  import cats.syntax.all._
  *
- * implicit val console = Console.sync[F]
- *
- * def myProgram[F[_]: Console]: F[Unit] =
- *   for {
- *     _ <- Console[F].println("Please enter your name: ")
- *     n <- Console[F].readLine
- *     _ <- if (n.nonEmpty) Console[F].println("Hello, " + n) else Console[F].errorln("Name is empty!")
- *   } yield ()
+ *  def myProgram[F[_] : Console : Monad]: F[Unit] =
```

Comment:
I _think_ this is the style enforced by our scalafmt, but I personally don't care :)

```suggestion
 *  def myProgram[F[_]: Console: Monad]: F[Unit] =
```

Reply:
Oh, you're right. 
I thought I've run `sbt prePR` to make sure it was in the right format. My bad, I accepted the commit

Reply:
I don't think scalafmt can check/format code in scaladocs, so it's not your fault :) thanks!